src/node_os.cc

```diff
@@ -189,14 +190,13 @@ static void GetInterfaceAddresses(const FunctionCallbackInfo<Value>& args) {
 
   int err = uv_interface_addresses(&interfaces, &count);
 
-  if (err == UV_ENOSYS)
-    return args.GetReturnValue().SetUndefined();
+  if (err == UV_ENOSYS) return;
 
   if (err) {
     CHECK_GE(args.Length(), 1);
-    env->CollectUVExceptionInfo(args[args.Length() - 1], errno,
-                                "uv_interface_addresses");
-    return args.GetReturnValue().SetUndefined();
+    USE(env->CollectUVExceptionInfo(
```

Comment:
Note.. the `CollectUVExceptionInfo` returns a `Maybe<void>` and we're returning right after either way. So the `USE(...)` here is just to squelch the compiler warning about the result not being checked directly... but we don't actually *need* to check the results here as it will still do the right thing.